subsys/power/debug.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2018,2020 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _ZEPHYR_SUBSYS_POWER_DEBUG_H_
+#define _ZEPHYR_SUBSYS_POWER_DEBUG_H_
+#include <zephyr.h>
+#include <kernel.h>
+
+#ifdef CONFIG_PM_DEBUG
+
+struct pm_debug_info {
+	uint32_t count;
+	uint32_t last_res;
+	uint32_t total_res;
+};
+
+static struct pm_debug_info pm_dbg_info[POWER_STATE_MAX];
+static uint32_t timer_start, timer_end;
```

Comment:
every file including this header will have its own copy of these variables. Shouldn't it be declared in C file and referenced as `extern` in the functions here ?

Reply:
plan is to get rid of this altogether, do not bother with that